src/EditorFeatures/Core/FindReferences/FindReferencesCommandHandler.cs

```diff
@@ -92,6 +89,7 @@ private bool TryExecuteCommand(int caretPosition, Document document)
             // or the host has no way to present results in a streaming manner.
             // Fall back to the old non-streaming approach to finding and presenting 
             // results.
+            var synchronousService = document.GetLanguageService<IFindReferencesService>();
```

Comment:
I don't quite understand this part. Didn't you just remove all the implementers of `IFindReferencesService`, or are there more now? (source.roslyn.io only shows the two deleted here).